eng/illink.targets

```diff
@@ -186,6 +209,8 @@
       <ILLinkArgs Condition="'$(ILLinkTrimXmlLibraryBuild)' != ''">$(ILLinkArgs) -x "$(ILLinkTrimXmlLibraryBuild)"</ILLinkArgs>
       <!-- don't remove the embedded substitutions xml resource since ILLink may run again on the assembly -->
       <ILLinkArgs Condition="'$(ILLinkSubstitutionsXml)' != ''">$(ILLinkArgs) --strip-substitutions false</ILLinkArgs>
+      <!-- don't remove the embedded link attributes xml resource since ILLink may run again on the assembly -->
+      <ILLinkArgs Condition="'$(ILLinkLinkAttributesXml)' != ''">$(ILLinkArgs) --strip-link-attributes false</ILLinkArgs>
```

Comment:
@marek-safar @vitek-karas @tlakollo - When we run the linker during the `dotnet/runtime` build, how does the linker know to ignore this LinkAttributes.xml file? We don't want to strip the attributes out during the `dotnet/runtime` build, but only when the app is being linked.

Reply:
`--strip-link-attributes false` & `--ignore-link-attributes true`